system/include/webgpu/webgpu.h

```diff
@@ -1017,6 +1068,7 @@ typedef struct WGPUComputePipelineDescriptor {
 
 typedef struct WGPUDeviceDescriptor {
     WGPUChainedStruct const * nextInChain;
+    char const * label;
```

Comment:
Needs to be implemented in the JS (see other implementations of descriptor labels for an example)